src/scss/components/_listings.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * Listing
+ * ===================================
+ * Example – SERP List – Restaurant Information
+ *
+ *
+ */
+
+$listing-default-borderColor: $color-border;
```

Comment:
Can likely get rid of the default here I reckon?  Makes sense as just `$listing-borderColor`